clients/android/app/src/main/java/app/lockbook/screen/FilesListFragment.kt

```diff
@@ -424,16 +435,32 @@ class FilesListFragment : Fragment(), FilesFragment {
                     }
 
                     icon.setImageResource(iconResource)
+
+                    itemView.setOnLongClickListener { view ->
+                        val popup = PopupMenu(view.context, view)
+
+                        popup.menu.add(0,1,0,"Remove")
+
+                        popup.setOnMenuItemClickListener { menuItem ->
+                            Lb.clearSuggestedId(item.fileMetadata.id)
+                            model.suggestedDocs.removeAt(i)
```

Comment:
rather than removing the item at the index, i would refetch suggested docs as a whole